### PR TITLE
NFID vault ancestor addresses

### PR DESCRIPTION
**Pull request by krzlabrdx**
…ible/location endpoint

### CODE REVIEW

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml [hunk 1]
```diff
@@ -3830,13 +3830,17 @@ components:
...
         - non_fungible_id
-        - last_updated_at_state_version
         - is_burned
+        - last_updated_at_state_version
       properties:
         non_fungible_id:
           $ref: "#/components/schemas/NonFungibleId"
         owning_vault_address:
           $ref: "#/components/schemas/Address"
+        owning_vault_parent_ancestor_address:
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:3840]: can we add short description what is that?

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml [hunk 2]
```diff
@@ -3830,13 +3830,17 @@ components:
...
+          $ref: "#/components/schemas/Address"
+        owning_vault_global_ancestor_address:
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:3842]: can we add short description what is that?

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml [hunk 3]
```diff
@@ -3830,13 +3830,17 @@ components:
       type: object
       required:
         - non_fungible_id
-        - last_updated_at_state_version
         - is_burned
+        - last_updated_at_state_version
       properties:
         non_fungible_id:
           $ref: "#/components/schemas/NonFungibleId"
         owning_vault_address:
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:3838]: can we add short description what is that? (i know you haven't touched it)

**Review by PawelPawelec-RDX — APPROVED** [review]

**Review by dhedey — APPROVED** [review]
Nice, thanks.

As Pawel says, some docs explaining what they mean would be useful :)

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml (hunk 1 above)
Comment (krzlabrdx) [reply src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:3841]: Added some short summarization.

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml (hunk 2 above)
Comment (krzlabrdx) [reply src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:3844]: Added some short summarization.

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml (hunk 3 above)
Comment (krzlabrdx) [reply src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:3838]: `last_updated_at_state_version` references `LastUpdatedAtStateVersion` that defines the appropriate description already.

**Pull request merged**